backend/src/test/java/io/debezium/configserver/CreateAndDeleteSqlSeverConnectorIT.java

```diff
@@ -74,7 +74,7 @@ public void testSqlServerDeleteConnectorFailed() {
     @Test
     public void testSqlServerDeleteConnectorSuccessful() {
         ConnectorConfiguration config = Infrastructure.getSqlServerConnectorConfiguration(1)
-                .with("database.dbname", "testdb");
+                .with("database.names", "testdb");
```

Comment:
nightly was only needed in the past to get in a change from 1.9.0 which was not yet released. We can go back with the [DBZ-5317](https://issues.redhat.com/browse/DBZ-5317) work to a stable version instead of nightly. Then switch to `2.0.0.Alpha3` images once we have 1.9 branch decoupled. wdyt?

Reply:
I think for `main`, we should explicitly test against `nightly`.  The whole point of those integration tests being executed in the UI action within the core repository in the first place is to try and identify PR failures in the core repository that have an impact on the UI.  If we explicitly test against say 2.0.0.Alpha3, then any future changes to core will not reflect any impacts and we'll be doing work after-the-fact rather than catching the problems when the code changes.